### ISSUE TITLE
container doesn't work

### ISSUE THREAD

**Issue by suportebeloj** [Alfresco/alfresco-docker-installer#100]
Hi, I followed all the steps in readm.md, but I can't log into /share.
I tried to access /alfresco but got a 404 error, I'm running in a ubuntu container on google cloud, but nothing works.

**Comment by aborroy** [comment]
This video may help you https://youtu.be/E_fvnlu3Lg0

**Comment by suportebeloj** [comment]
Right, I checked the logs folder, and from what I saw the alfresco doesn't create any logs in the folder, I believe I have no writing rights for it.

I dumped the container log, I don't know if it will serve as a sampling.

In addition, I also uploaded it to a recording on my machine where I redid the installation from scratch and got the same problem.

[docker-compose-logs-alfresco](https://drive.google.com/file/d/1bz4sqrVXPHr5OM9Ot6yNEmwf0G4_coPh/view?usp=sharing)

[youtube video](https://youtu.be/zzbLoIlBFc4)
